.github/workflows/tests.yml

```diff
@@ -17,76 +15,28 @@ jobs:
     name: ${{ matrix.lisp }} on ${{ matrix.os }}
     strategy:
       matrix:
-        # Use ccl-bin/1.12.1 instead of 'ccl' because of
-        # https://github.com/roswell/roswell/issues/534.
-        # TODO: Revert when Roswell is functional again.
-        lisp: [sbcl-bin, ccl-bin/1.12.1]
-        rosargs: [dynamic-space-size=3072]
-        os: [ubuntu-latest, macos-latest] # try windows-latest when we understand commands to install Roswell on it
```

Comment:
@aadcg Why remove macos and CCL?

Reply:
I don't know whether the work was being put in the spotlight. I don't think Pierre is questioning it, but rather trying to catch up. I could be however mistaken, and will not speak for Pierre.

With regards to necroposting, I agree with Andre, let us please refrain from doing so. It breaks our inertia and has us justifying engineering decisions which happened a while ago. If you want to change something, then I would suggest proposing a new topic espousing the virtues of a richer testing matrix.

Personally, I am not in favor of a more complex matrix. I also agree that we should be mastering one single cell within the matrix before introducing new cells.